README.md

```diff
@@ -100,7 +128,11 @@ ialirt-data-access --api-key <your-api-key> --url https://ialirt.imap-mission.co
 
 Example:
 ```bash
-ialirt-data-access --api-key <api_key> --url https://ialirt.imap-mission.com/api-key ialirt-db-query --met_start 100
+ialirt-data-access --api-key <api_key> --url https://ialirt.imap-mission.com/api-key space-weather --instrument <instrument>
+```
+An equivalent curl command would be:
+```bash
+curl -H "x-api-key: $IALIRT_API_KEY" "https://ialirt.imap-mission.com/api-key/space-weather?instrument=mag"
```

Comment:
In the earlier sections you are prefixing these shell commands with a dollar sign `$ curl`, but in this section you aren't. I'd just be consistent one way or the other.